### DISCUSSION TITLE
Use own segmented dataset for MAE training

### DISCUSSION THREAD

**Discussion by nicekwow** [open-mmlab/mmclassification#1480] [Q&A]
I would like to know what dataset format to use for self supervised training using segmented datasets

**Comment by fangyixiao18** [comment]
Can this [doc](https://mmpretrain.readthedocs.io/en/latest/notes/pretrain_custom_dataset.html) help you?
If not, could u please describe your question more precisely? What did you mean of using segmented datasets

**Reply by nicekwow** [reply]
What I mean is that my dataset is a annotated segmented dataset, consisting of the original image and its corresponding annotated mask image. So, what form do I need to convert it into to use MAE

**Reply by fangyixiao18** [reply]
> What I mean is that my dataset is a annotated segmented dataset, consisting of the original image and its corresponding annotated mask image. So, what form do I need to convert it into to use MAE

You could use `CustomDataset`, and the `data_prefix` is the path of your original images. Please check the [document](https://mmpretrain.readthedocs.io/en/latest/notes/pretrain_custom_dataset.html).
